packages/ui/src/components/Designer/index.tsx

```diff
@@ -27,6 +28,12 @@ import Root from '../Root';
 import ErrorScreen from '../ErrorScreen';
 import CtlBar from '../CtlBar';
 
+const px2mm = (px: number): number => {
```

Comment:
we already have px2mm in the image schema, should we add it to the common/helper.ts file where the other conversion functions are?

Reply:
Ah, yes. It should be.